_stbt/ocr.py

```diff
@@ -622,29 +629,54 @@ def _find_tessdata_dir():
     raise RuntimeError('Installation error: Cannot locate tessdata directory')
 
 
-def _log_ocr_image_debug(imglog, text):
+def _log_ocr_image_debug(imglog, output=None):
     if not imglog.enabled:
         return
 
+    if imglog.name == "ocr":
+        title = "stbt.ocr"
+        match_text = False  # pylint:disable=redefined-outer-name
+    else:
+        match_text = True
+        result = imglog.data["result"]
+        title = "stbt.match_text(%r): %s" % (
+            imglog.data["text"],
+            "Matched" if result else "Didn't match")
+        hocr = imglog.data["hocr"]
+        if hocr is None:
+            output = u""
+        else:
+            output = u"".join(x for x, _ in _hocr_iterate(hocr))
+
     template = u"""\
-        <h4>OCR</h4>
+        <h4>{{title}}</h4>
 
         <div class="annotated_image">
           <img src="source.png" />
-          {{ draw(roi, image_region, "roi", text) }}
+          {{ draw(roi, image_region, "roi") }}
+          {% if match_text %}
+          {{ draw(result.region, image_region, result.match,
+                  title=result.text) }}
```

Comment:
It would be interesting to draw all the text that hocr read here, not just the matched region.